### ISSUE TITLE
Autosummary generates overly-long summaries if first line doesn't end with a period

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1374]
If a function/module/etc has a docstring that looks like this:

```
I am the summary

...but it didn't end with a period, and there IS a sentence that ends with a period later, so this whole thing will become the summary.
```

...then the entire thing will get included as the "summary".  This was introduced by the fix to issue 943 - #943 - which attempted to detect situations where the first sentence of a docstring spanned multiple lines.

Unfortunately, it doesn't include detection for situations like the one above, where blank lines are used to indicate the end of the "summary".

I have a patch, which detects blank lines, and will submit a pull request shortly...

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1374
- Originally reported by: [barnabas79](https://bitbucket.org/barnabas79)
- Originally created at: 2014-02-04T04:20:29.821

**Comment by shimizukawa** [comment]
_From [barnabas79](https://bitbucket.org/barnabas79) on 2014-02-04 03:43:09+00:00_

Created a pull request:
https://bitbucket.org/birkenfeld/sphinx/pull-request/218/closes-1374-autosummary-generates-overly/diff

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-01 07:28:31+00:00_

Merged in barnabas79/sphinx (https://bitbucket.org/birkenfeld/sphinx/pull-request/218)

Closes #1374: Autosummary generates overly-long summaries if first line doesn't end with a period

→ <<cset 1f4016d679924ffd8f098065374c80454c25704b>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-01 07:28:31+00:00_

Closes #1374: Autosummary generates overly-long summaries if first line doesn't end with a period

→ <<cset 05f1049b3ec386f32d89f601403a1fa293b86c68>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-03-01 07:42:04+00:00_

Closes #1374: Autosummary generates overly-long summaries if first line doesn't end with a period

→ <<cset 05f1049b3ec386f32d89f601403a1fa293b86c68>>
